packages/@okta/vuepress-theme-prose/components/Quickstart.vue

```diff
@@ -1,265 +1,87 @@
-
-      <div class="quickstart--content">
-        <Content slot-key="intro" />
-
-        <Content slot-key="account" />
-
-        <Content slot-key="client" />
-        <p>Now, let's add authentication to your client. (Already done? <a href="#server_setup" @click.prevent="scrollToServer()">Skip to server setup</a>)</p>
-        <div class="code-selector" id="client-selector">
-          <h3>Client</h3>
-          <ul>
-            <li v-for="(client, index) in $themeConfig.quickstarts.clients" class="with-icon" :key="index">
-              <a :class="{active: activeClient==client.name}" @click="activeClient=client.name">
-                <i :class="['with-icon','icon', 'code-'+(client.codeIconName || client.name)+'-32']"></i>{{ client.label }}
-              </a>
-            </li>
-          </ul>
-        </div>
-
-        <Content :pageKey="activeClientComponent" id="client_content" class="example-content-well"></Content>
-
-
-        <Content slot-key="server" />
-
-        <div class="code-selector" id="server-selector">
-          <h3>Server</h3>
-          <ul>
-            <li v-for="(server, index) in $themeConfig.quickstarts.servers" class="with-icon" :key="index">
-              <a :class="{active: activeServer==server.name}" @click="activeServer=server.name">
-                <i :class="['with-icon','icon', 'code-'+(server.codeIconName || server.name)+'-32']"></i>{{ server.label }}
-              </a>
-            </li>
-          </ul>
-        </div>
-
-        <div class="code-selector" id="framework-selector">
-          <h3>Framework</h3>
-          <ul>
-            <li v-for="(framework, index) in activeServerFrameworks" :key="index">
-              <a :class="{active: activeFramework==framework.name}" @click="activeFramework=framework.name">
-                {{ framework.label }}
-              </a>
-            </li>
-          </ul>
-        </div>
-
-        <Content :pageKey="activeServerComponent" id="server_content" class="example-content-well"></Content>
-      </div>
-
-    </div>
-    <div class="on-this-page">
-      <OnThisPage :items="otpItems"/>
-    </div>
-
-  </div>
-</template>
-
 <script>
   export default {
-    components: {
-      Sidebar: () => import('./Sidebar.vue'),
-      PageTitle: () => import('./PageTitle.vue'),
-      OnThisPage: () => import('./OnThisPage.vue')
-    },
+
     data() {
       return {
-        activeTab: null,
         currentHash: null,
         activeClient: null,
-        activeClientComponent: null,
         activeServer: null,
-        activeServerComponent: null,
         activeFramework: null,
-        otpItems: [
-          {'title': 'Create an Account', 'path': '#create-an-okta-account'},
-          {'title': 'Client Setup', 'path': '#client-setup'},
-          {'title': 'Server Setup', 'path': '#server-setup'}
-        ]
+        mappings: {
+          '#/okta-sign-in-page/nodejs/express' : '/docs/guides/sign-into-web-app/nodeexpress/before-you-begin/',
```

Comment:
Not suggesting this, just putting it out there - if this page just booted you to a url that had all the elements from the hash but didn't use a hash (e.g. /quickstarts/#/okta-sign-in-page/nodejs/express => /quickstarts/okta-sign-in-page/nodejs/express) then we could have the actual redirects in conductor.yml so they'd have the same management.

Downsides: 
- two redirects instead of one
- conductor.yml may be consolidated and consistent, but that doesn't make it desirable
- you might have to create any intermediate directories for the conductor-file-based approach to work (you could resolve this by not having more paths in the original redirect). 

I'm happy with this approach, but I wanted to make sure the alternative was considered.